### DISCUSSION TITLE
Can I get some actors in my plotter instance directly?

### DISCUSSION THREAD

**Discussion by Lemonlee8** [pyvista/pyvista#3588] [Q&A] [answered]
I am using pyvistaqt QtInteractor as the central widget of Qt application. And I want to get all of the actors, also some of them, in my current plotter( an instance of class QtInteractor). Is there any ready-made method to get around this?

Any help on the topic is appreciated.

**Comment by Lemonlee8** [comment, marked as answer]
For example, if I create a Plotter object and then add a sphere mesh to it, I also give a name 'sphere' to the new actor. What I want to know is how can I get this actor object by the name 'sphere'. Very thanks.

```python
import pyvista as pv
pl = pv.Plotter()
actor = pl.add_mesh(pv.Sphere(),name='sphere')
dss=pl._datasets
pl.show()
```

**Reply by banesullivan** [reply]
`pl.renderer.actors` tracks all actors added to the renderer with PyVista.  To get an actor by name, simply: `pl.renderer.actors[name]`

Ref https://docs.pyvista.org/api/plotting/_autosummary/pyvista.Renderer.actors.html#actors

**Comment by Lemonlee8** [comment]
Very thanks, it works..
